### PR TITLE
trapz

### PR DESCRIPTION
**Pull request by shireenchand**
Added pytorch trapz frontend and tests

### CODE REVIEW

**Comment by shireenchand** [comment]
#11971

**Comment by aditya-hari** [comment]
Hey! Not really sure about the test, a conditional test doesn't really make a lot of sense. You might have to create your own sampling strategy for this one.

**Comment by shireenchand** [comment]
@aditya-hari can you please suggest me a good way of implementing that?

**Comment by aditya-hari** [comment]
Try getting the axis using dtype_values_axis rather than a simple st.integers()  
And for the general structure of the test, you can maybe try something similar to the test for torch.where, which uses a sampled boolean for the conditional
